mrp_subcontracting_resupply_status/views/purchase_order_views.xml

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <record id="purchase_order_form_resupply_status" model="ir.ui.view">
+        <field name="name">purchase.order.form.resupply.status</field>
+        <field name="model">purchase.order</field>
+        <field name="inherit_id" ref="purchase.purchase_order_form" />
+        <field name="arch" type="xml">
+            <field name="invoice_status" position="after">
```

Comment:
maybe better to put it before the receipt status in order to follow the order of the flow (first resupply, then receive, finally invoice)